package.json

```diff
@@ -59,6 +59,9 @@
     "gulp-util": "^3.0.7",
     "gzip-js": "~0.3.2",
     "jest": "^15.1.1",
+    "jest-cli": "^15.1.1",
```

Comment:
jest seemingly just delegates to jest-cli but it does have the same exports so (y).